app/views/budgets/_form.html.erb

```diff
@@ -1,5 +1,5 @@
-<%= simple_form_for @budget do |f| %>
+<%= simple_form_for budget, url: budgets_path do |f| %>
```

Comment:
Is specifying the `url` needed here?

Reply:
No, thanks for the pick up, remove this one and the one in the users edit form as well, ad-css-styling sha: 0513b48